framework/include/fvkernels/FVDiffusion.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "FVFluxKernel.h"
+
+class FVDiffusion : public FVFluxKernel
+{
+public:
+  static InputParameters validParams();
+  FVDiffusion(const InputParameters & params);
+
+protected:
+  virtual ADReal computeQpResidual() override;
+
+  const ADMaterialProperty<Real> & _coeff_left;
+  const ADMaterialProperty<Real> & _coeff_right;
```

Comment:
For FV, it is easy to understand if we are using `left` and `right`  because we are working on a face.  Is it feasible to change DG terminologies  to `left` and `right`?

I would like to keep using `left` and `right`. `element` and `neighbor` do not have much information

Reply:
Especially awkward when discussing the "element element" vs the "neighbor element" since they are both elements.  In MOOSE there are some places where "element" just means generically "an element" (i.e. not side-specific) and other places where "element" means the "not neighbor" side.  And I also wanted to convey the fact that neither side is really a privileged side.  But yes - it's also not ideal that we have other *loosely* followed conventions in moose.

Reply:
> And I also wanted to convey the fact that neither side is really a privileged side.

But left is privileged. The normal points from left to right.

Reply:
We could choose to orient the normal out from the right if we wanted - that orientation is just an artifact of an optimization - where we only calculate the numerical flux once for the contribution to each neighboring element.  We could call computeQpResidual once with the normal pointing outward for each the left and right elements, but that would be wasteful.  So at a fundamental level, the left isn't really privileged.

Reply:
> But left is privileged. The normal points from left to right.

Again this statement is true only if the element on the "left" has a lower ID...

Reply:
Yeah - I guess my real gripe is that `element` is really confusing and overloaded here.  I don't really care what we do, except that I don't like having to wonder whether `element` is referring to something related to two-sides of a face or not.